packages/sdk/src/controllers/FrameController.ts

```diff
@@ -383,6 +384,23 @@ export class FrameController {
         return res.addFrame(type, x, y, width, height).then((result) => getEditorResponseData<Id>(result));
     };
 
+    /**
+     * This method will create a new barcode frame of 'type' type to the layout positioned on the requested
+     * coordinates. Any coordinate that is not specified will default to 'center'. If any size property is
+     * not specified, that property will use a sensible default
+     *
+     * @param type the type of barcode to create
+     * @param x optional X coordinate of the new frame within the layout
+     * @param y optional Y coordinate of the new frame within the layout
+     * @param width optional Width of the new frame within the layout
+     * @param height optional Height of the new frame within the layout
+     * @returns
+     */
+    createBarcodeFrame = async (type: BarcodeType, x?: number, y?: number, width?: number, height?: number) => {
```

Comment:
Given that all these properties are optional I would suggest the following:

```suggestion
    createBarcodeFrame = async (type: BarcodeType, position?: { x?: number, y?: number, width?: number, height?: number }) => {
```